roles/ceph-common/tasks/installs/install_on_suse.yml

```diff
@@ -19,6 +19,7 @@
   with_items: "{{ suse_package_dependencies }}"
   register: result
   until: result is succeeded
+  when: ansible_python_version is version( '3', '<')
```

Comment:
Looks like this task is a duplicate

Reply:
Should I remove this task completely (in another PR to not mix things up)?

Reply:
See #4349